Minio/Helper/s3utils.cs

```diff
@@ -35,22 +37,23 @@ internal static bool IsAmazonEndPoint(string endpoint)
         // the China (Beijing) Region. Customers with existing AWS credentials
         // will not be able to access resources in the new Region, and vice versa.
         // For more info https://aws.amazon.com/about-aws/whats-new/2013/12/18/announcing-the-aws-china-beijing-region/
-        internal static bool IsAmazonChinaEndPoint(string endpoint)
+        public static bool IsAmazonChinaEndPoint(string endpoint)
+
         {
 
             return endpoint == "s3.cn-north-1.amazonaws.com.cn";
         }
         // IsGoogleEndpoint - Match if it is exactly Google cloud storage endpoint.
-        internal static bool IsGoogleEndPoint(string endpoint)
+        public static bool IsGoogleEndPoint(string endpoint)
         {
             return endpoint == "storage.googleapis.com";
```

Comment:
I remember google storage supports signature v2.  does minio-dotnet support signaure v2 as well?
I remember google storage does not support virtual style endpoint for buckets.  If so why do we need to treat it as specially?